src/NUnitFramework/tests/Constraints/PredicateConstraintTests.cs

```diff
@@ -52,5 +55,46 @@ public void CanUseConstraintExpressionSyntax()
         {
             Assert.That(123, Is.TypeOf<int>().And.Matches<int>((int x) => x > 100));
         }
+
+        [TestCase(typeof(object))]
+        [TestCase(typeof(string))]
+        [TestCase(typeof(int?))]
+        [TestCase(typeof(AttributeTargets?))]
+        public static void ActualMayBeNullForNullableTypes(Type type)
+        {
+            // https://github.com/nunit/nunit/issues/1215
+            var methodInfo = typeof(PredicateConstraintTests)
+                .GetMethods(BindingFlags.NonPublic | BindingFlags.Static)
+                .Single(method => method.Name == nameof(ActualMayBeNullForNullableTypes) && method.GetParameters().Length == 0)
```

Comment:
Isn't the test for the number of parameters redundant since there is only one matching method?

Reply:
Yes. I see I overlooked that while juggling different approaches. The compatibility extension methods are far from complete; sometime I am going to flesh them out.

Reply:
I only noticed because GetParameters cannot be used on an open generic method in CF.  Since it was redundant anyway, I just removed it in my build.

Reply:
Oh, very interesting!